CMakeLists.txt

```diff
@@ -10,6 +10,15 @@ project(llvm-dialects)
 
 message(STATUS "Begin configuring llvm-dialects...")
 
+function(set_compiler_options PROJECT_NAME)
```

Comment:
Maybe name this enable_compiler_color_output? This is not really setting anything more than that